5-network/05-fetch-crossorigin/article.md

```diff
@@ -377,11 +377,11 @@ Additionally, to grant JavaScript access to any response headers except `Cache-C
 
 **For unsafe requests, a preliminary "preflight" request is issued before the requested one:**
 
-- → The browser sends `OPTIONS` request to the same URL, with headers:
+- → The browser sends an `OPTIONS` request to the same URL, with the headers:
     - `Access-Control-Request-Method` has requested method.
     - `Access-Control-Request-Headers` lists unsafe requested headers.
-- ← The server should respond with status 200 and headers:
+- ← The server should respond with status 200 and the headers:
     - `Access-Control-Allow-Methods` with a list of allowed methods,
     - `Access-Control-Allow-Headers` with a list of allowed headers,
-    - `Access-Control-Max-Age` with a number of seconds to cache permissions.
-- Then the actual request is sent, the previous "safe" scheme is applied.
+    - `Access-Control-Max-Age` with a number of seconds to cache the permissions.
+- Then the actual request is sent and the previous "safe" scheme is applied.
```

Comment:
Is the comma needed before the `and`?

Reply:
> Is the comma needed before the `and`?

I'd say both is correct. With and without comma. See https://www.grammarly.com/blog/comma-before-and/

Reply:
I'm reading the link, and it says that the comma is needed.

I suppose, we have the conjunction here. Two independent sentences.

`The word and is a conjunction, and when a conjunction joins two independent clauses, you should use a comma with it`.

Reply:
Added a comma